infra/cifuzz/cifuzz.py

```diff
@@ -311,7 +312,8 @@ def get_files_covered_by_target(latest_cov_info, target_name, src_in_docker):
     ValueError: When the src_in_docker is not defined.
   """
   if not src_in_docker:
-    raise ValueError('Project base is not defined. Can\'t get coverage')
+    logging.error('Project base is not defined. Can\'t get coverage')
```

Comment:
what's the reason for this change?

Reply:
On all other errors/issues I have been returning None so I wanted to maintain consistency. I also would just end up catching it in the keep_affected_fuzzers method and then doing the same thing as if it returned None.

Reply:
nit: what is "project base"? can the message be more descriptive?

and for the coverage part, let's also be more explicit, e.g. "Can't get coverage information from OSS-Fuzz."

Reply:
Fixed.